docs/comparison_template_library.md

```diff
@@ -0,0 +1,27 @@
+---
+tags:
+  - API
+  - comparisons
+  - Date Comparison
+---
+
+# Documentation for `comparison_template_library` 
+
+The `comparison_template_library` contains pre-made comparisons available for use directly [as described in this topic guide](../topic_guides/customising_comparisons.html#method-1-using-the-comparisonlibrary).
+However, not every comparison is available for every [Splink-compatible SQL backend](../topic_guides/backends.html).
```

Comment:
Once this is merged we can raise an issue to include a generated table a la `cl` and `cll` here also